package.json

```diff
@@ -26,7 +26,7 @@
     "psql": "pnpm -F commonwealth psql",
     "reset-db": "pnpm -F commonwealth reset-db",
     "sanity": "chmod u+x scripts/sanity.sh && ./scripts/sanity.sh",
-    "start": "pnpm -F commonwealth start",
+    "start": "chmod u+x scripts/start.sh && ./scripts/start.sh",
```

Comment:
I'm not sure we want to make `pnpm start` from root start all services. Starting everything is resource hungry and not recommended to run for simple development. Additionally, starting these services requires RabbitMQ and Redis to be up as well so making it the default doesn't seem right.

Reply:
it only starts all services when first arg is "all", the default is just starting the server

Reply:
ah ok didn't see that